benchmark/sparse/linear_regression/mxnet_sparse_model.py

```diff
@@ -6,35 +6,50 @@
 import mxnet as mx
 
 
-def run_benchmark(train_data, train_label, eval_data, eval_label, batch_size, epochs):
-    train_iter = mx.io.NDArrayIter(train_data, train_label, batch_size=batch_size, shuffle=True,
-                                   label_name='lin_reg_label', last_batch_handle='discard')
-    eval_iter = mx.io.NDArrayIter(eval_data, eval_label, batch_size=batch_size, shuffle=False,
-                                  label_name='lin_reg_label', last_batch_handle='discard')
+def run_benchmark(train_data, train_label, eval_data, eval_label, batch_size, epochs, start):
+    train_iter = mx.io.NDArrayIter(train_data, train_label, batch_size, last_batch_handle='discard', label_name='label')
+
+    eval_iter = mx.io.NDArrayIter(eval_data, eval_label, batch_size=batch_size,
+                                  label_name='label', last_batch_handle='discard')
 
     X = mx.sym.Variable('data', stype='csr')
-    Y = mx.symbol.Variable('lin_reg_label', stype='csr')
-    fully_connected_layer = mx.sym.FullyConnected(data=X, name='fc1', num_hidden=10)
-    lro = mx.sym.LinearRegressionOutput(data=fully_connected_layer, label=Y, name="lro")
+    weight = mx.symbol.Variable('weight', stype='row_sparse', shape=(train_data.shape[1], 1))
+    pred = mx.sym.sparse.dot(X, weight)
```

Comment:
please add bias here according to example:
https://mxnet.incubator.apache.org/tutorials/sparse/train.html

in keras, when you add Dense layer, bias is added by default

Reply:
Bias is added in Keras only if present. We dont need to add bias explicitly. As I am not adding bias in the MXNet model, its not added in the keras model either - https://github.com/awslabs/keras-apache-mxnet/blob/master/keras/layers/core.py#L875

Reply:
added